packages/components/input/src/input.js

```diff
@@ -0,0 +1,52 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './input.css';
+
+customElementDefineHelper(
+	'ts-input',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				disabled: { type: Boolean, attribute: 'disabled', reflect: true }
+			};
+		}
+
+		attributeChangedCallback(name, oldval, newval) {
+			const input = this.getSlotInput();
+			if (name === 'disabled' && input) {
+				input.disabled = newval;
+			}
+			super.attributeChangedCallback(name, oldval, newval);
+		}
+
+		firstUpdated(changeedProperties) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "changeedProperties". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)